src/astroFactory.js

```diff
@@ -0,0 +1,81 @@
+import { MeshBuilder } from "@babylonjs/core/Meshes/meshBuilder"
+import { Texture } from "@babylonjs/core/Materials/Textures/texture"
+import { StandardMaterial } from "@babylonjs/core/Materials/standardMaterial"
+import { Scalar } from "@babylonjs/core/Maths/math.scalar"
+import { Animation } from "@babylonjs/core/Animations/animation"
+
+import logger from "./logger";
+
+import distortTexture from "../assets/textures/distortion.png";
+import rockTextureN from "../assets/textures/rockn.png";
+import rockTexture from "../assets/textures/rock.png";
+
+class AstroFactory {
+    static createPlanet(opts, scene) {
```

Comment:
instead of static functions in a class you can export them as standalone functions outside of the context of a class.

Reply:
are there any benefits to leaving them in a class or to moving them outside of a class, or is it more a matter of preference/readability? I like the encapsulation of a Class, but it may not really add anything meaningful